pkgs/by-name/mi/millisecond/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  python313,
+  gobject-introspection,
+  gtk4,
+  desktop-file-utils,
+  appstream,
+  glib,
+  wrapGAppsHook4,
+  libadwaita,
+}:
+
+let
+  pythonEnv = python313.withPackages (p: [
+    p.pygobject3
+  ]);
+in
+
+stdenv.mkDerivation rec {
+  pname = "millisecond";
+  version = "v0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "gaheldev";
+    repo = "Millisecond";
+    tag = version;
+    hash = "sha256-SMGcSlbOfBX5gAwB7CaHRthf9EN5QWAN9ZzrcbQXtm8=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    ninja
+    pythonEnv
+    desktop-file-utils
+    appstream
+    glib
+    pkg-config
+    gtk4
+    libadwaita
+    gobject-introspection
+  ];
+
+  meta = {
+    homepage = "https://github.com/gaheldev/Millisecond";
+    description = "Optimize your Linux system for low latency audio";
+    mainProgram = "millisecond";
+    license = lib.licenses.gpl3;
```

Comment:
`lib.licenses.gpl3` is deprecated, please update to `lib.licenses.gpl3Plus`, which is the license used by the project